common/src/main/java/tachyon/network/protocol/RPCBlockReadResponse.java

```diff
@@ -34,6 +34,15 @@
   private final DataBuffer mData;
   private final Status mStatus;
 
+  /**
+   * Constructs a new RPC response of a {@link RPCBlockReadRequest}.
+   *
+   * @param blockId the id of the block
+   * @param offset the block offset to begin reading at
```

Comment:
Since this class is a response, we should switch to the past tense, e.g. `offset the block offset that the read began at` and `length the number of bytes read`